src/sass/app_2/_common/base/typography.scss

```diff
@@ -28,6 +28,13 @@
     @return calc(#{$a*100}vw #{$sign} #{$b});
 }
 // Indrek Pass also has a fantastic mixin here: http://sassmeister.com/gist/7f22e44ace49b5124eec
+html {
+    text-shadow: 1px 1px 1px rgba(0, 0, 0, 0.004); //stylelint-disable-line number-max-precision
+    text-rendering: optimizeLegibility !important;
```

Comment:
does this mess with translations? I remember something similar messing up some special characters in some other languages

Reply:
We have the same `optimizeLegibility` rule in `binary-style` typography as well. 
It shouldn't affect translations. Same goes with `text-shadow`.